tests/testthat/test-options.R

```diff
@@ -33,3 +33,16 @@ test_that("bcdata.single_download_limit", {
   )
 
 })
+
+test_that("bcdata.single_download_limit can be changes",{
+  lt <- withr::local_options(list(bcdata.single_download_limit = 13),
+                       getOption("bcdata.single_download_limit"))
+  expect_equal(lt$bcdata.single_download_limit, 13)
```

Comment:
```suggestion
  withr::local_options(list(bcdata.single_download_limit = 13))
                       
  expect_equal(getOption("bcdata.single_download_limit"), 13)
  expect_equal(bcdc_single_download_limit(), 13)
```

Reply:
I think that should do it. `withr::local_options()` sets the options for the current scope (the test) so you can test their effect, then when the test exits, it resets it.